configure

```diff
@@ -268,12 +268,10 @@ if test "$gcc" -eq 1 && ($cc $CFLAGS -c $test.c) >> configure.log 2>&1; then
       if test "${uname}" = "eabi"; then
         uname=arm
       fi
-      if test $buildacle -eq 1; then
-        if test $native -eq 0; then
-          ARCH=armv8-a+crc
-        else
-          ARCH=native
-        fi
+      if test $native -eq 0; then
+        ARCH=armv7-a
+      else
+        ARCH=native
```

Comment:
This block has nothing to do with arm-eabi... The previous block just sets uname to "arm" because eabi toolchain doesn't set it to recognizable value.

Reply:
Although it set uname to arm, but it goes to the wrong ARCH branch if built with --acle. It goes to  armv8-a+crc which is wrong!

Reply:
We only allow enabling ACLE on armv8-a... NEON can be enabled on ARMv7 also...